mantidproject/mantidimaging
# Documentation updates

See commit messages.

To test:
- `python setup.py docs_api`
- `python setup.py docs`
- See that changed docs make sense
